.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+(Thanks for sending a pull request! Please fill out the blanks below.)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-file-name-irregular-characters] Do not use `_` in a file name](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4576380)